tket/tests/test_TwoQubitCanonical.cpp

```diff
@@ -129,7 +157,7 @@ SCENARIO("Testing two-qubit canonical forms") {
     circ.add_op<unsigned>(tket::OpType::Rz, 0.5, {1});
     circ.add_op<unsigned>(tket::OpType::Rz, 0.5, {0});
     circ.add_op<unsigned>(tket::OpType::Rx, 1.2, {1});
-    Eigen::Matrix4cd U = get_matrix_from_2qb_circ(circ);
+    Eigen::Matrix4cd U = tket_sim::get_unitary(circ);
```

Comment:
This is actually changing the test (a different function). Does the function `get_matrix_from_2qb_circ` still work? Should do, surely! If so, could you add a check that the unitaries produced are almost identical?

Reply:
My thinking was to use `tket_sim::get_unitary` everywhere possible in the tests, for consistence. That `get_matrix_from_2qb_circ` behaves correctly is already checked in the other tests I added (lines 47-72).